### ISSUE TITLE
2.10.7 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3571]
- #3502 
- #3519 
- #3545
- #3499 
- #3446 
- #3583

## Pick list
```
git cherry-pick 361f050fccdbd5d1a8f39145e8e3b23465e22e99
git cherry-pick c2fe0f91281e70590a45bc1ae1f189411d1e5016
git cherry-pick 747de76b53d19f96ce93e4bbf3c9aa28f4f00c72
git cherry-pick 353b8cd5d9233a891637667f99a61fecbc7add92
git cherry-pick 6663cf5c02017244028625506bb898c32819af52
```

**Comment by joeparsons** [comment]
## Release note snippet

`git log 2.10.6..HEAD --pretty='format:- %s'`
```
- Closes #3445 Make config dependencies, module dependencies in az_paragraphs:az_paragraphs_view (#3446)
- Closes #3498 Make config dependencies, module dependencies in az_seo. (#3499)
- Fixes #3544 AZBackgroundMediaFormatter Breaks with Missing Files (#3545)
- Closes #3453 | Remove duplicate required asterisk on Webforms (#3519)
- Close #3501 remove logging code from az_course importer. (#3502)
- Prepare 2.10.x branch for new minor release branch 2.11.x (#3528)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.10.7
